addons/website/tests/test_ui.py

```diff
@@ -645,6 +645,12 @@ def test_website_no_dirty_lazy_image(self):
 
```

Comment:
Commit message:
The commit title exceeds 72 char => if you change "create" by "add" it's all good :+1: